File: Recursion/fizzbuzz/fizzbuzz.py


def fizzBuzz(num):
    if num <= 0:
        return;

    if num % 3 == 0 and num % 5 == 0:
        print("FizzBuzz")

    elif num % 3 == 0:
        print("Fizz")

    elif num % 5 == 0:
        print("Buzz")

    else:
        print(num)
    
    num -= 1
    fizzBuzz(num)



fizzBuzz(35)

File: Recursion/fizzbuzz/fizzbuzz.js


function fizzBuzz(num){
    if (num <= 0) return;
    else if (num % 3 === 0 && num % 5 === 0) console.log("FizzBuzz");
    else if (num % 3 === 0) console.log("Fizz");
    else if (num % 5 === 0) console.log("Buzz");
    else console.log(num);
    fizzBuzz(num-1)
}

fizzBuzz(35)
